spec/support/faker.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.configure do |config|
+  config.after do
+    Faker::UniqueGenerator.clear
```

Comment:
🗒️ If you have a bunch of tests that create data using unique faker values like `Faker::Finance.unique.ticker`, faker's pool of unique values can eventually get exhausted. This line resets the pool after each spec so that this is much less likely to happen.